qml/ActionList.qml

```diff
@@ -28,7 +28,7 @@ Column {
             var info = { name: name, path: "plugins/" + name + ".qml" }
             plugins.append(info)
         }
-        var names = [ "RestartNetwork", "RestartUI",
+        var names = [ "RestartBluetooth", "RestartNetwork", "RestartUI",
                       "CleanPackageCache", "CleanTracker" ]
```

Comment:
```suggestion
        var names = [ "RestartNetwork", "RestartUI",
                      "CleanPackageCache", "CleanTracker",
                      "RestartBluetooth" ]
```
My feeling is we should probably add new entries to the end, to avoid disrupting the existing item location. What do you think?

Reply:
As it is it has all the restarts in alphabetic order, and the cleanups after. 

I think it's better UI to have logically similar things grouped together, as opposed to having another  "Restart" option appearing at the end after the "Cleanups".